Chisel/Makefile

```diff
@@ -0,0 +1,11 @@
+PREFIX ?= /usr/local/lib
+
+install:
+	xcodebuild \
+		-scheme Chisel \
+		-configuration Release \
+		-sdk iphonesimulator \
+		install \
+		DSTROOT="$(PREFIX)" \
+		INSTALL_PATH=/ \
```

Comment:
According to the docs, the above two should be flipped. `DSTROOT=/` and `INSTALL_PATH=$(PREFIX)`.